src/main.rs

```diff
@@ -171,15 +171,13 @@ pub async fn do_main() -> Result<()> {
 	} else {
 		cfg.ipfs_port.0
 	};
-
-	let seed = if cfg.ipfs_seed_randomize {
-		let seed: u64 = thread_rng().gen();
-		info!("Using random seed: {seed}");
-		seed
-	} else {
-		cfg.ipfs_seed
+	let seed = match cfg.ipfs_seed.clone() {
```

Comment:
Do we have the case where the seed is set to 'random'? Do we even need it?

Reply:
well AFAIK the random is used strictly for docker and that was the reason why I switched to Option<u64> to string

Reply:
I'm not sure if we can go with some "0" value to avoid having different type for seed (instead of "random"). Do we need to discuss this?